core/src/main/java/com/netflix/conductor/core/execution/DeciderService.java

```diff
@@ -399,16 +397,16 @@ void checkForTimeout(TaskDef taskDef, Task task) {
     }
 
     @VisibleForTesting
-    boolean isResponseTimedOut(TaskDef taskDefinition, Task task) {
+    void checkResponseTimeOut(TaskDef taskDefinition, Task task) {
```

Comment:
better to move the logic that times out the task out of this method into its own method so the code is readable, we won't be able to read above method and know that the task could be timed out without diving into checkResponseTimeOut method, for example, the following code is more readable:

```
if (isResponseTimedOut(taskDefinition, pendingTask) {
   timeoutTask(pendingTask);
}
```